DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+<figure>
+	<img src="Images/Timing/IndexedAddressing.png" />
+	<figcaption>Indexed addressing is based on an index segment that references all [=media segments=].</figcaption>
+</figure>
+
+The MPD defines the byte range of the track file that contains the index segment. The index segment informs the client of all the [=media segments=] that exist, the time spans they cover on the [=sample timeline=] and their byte ranges.
+
+Multiple representations SHALL NOT be stored in the same track file.
+
+At least one `Representation/BaseURL` element SHALL be present in the MPD and SHALL contain a reference to the track file.
+
+The `SegmentBase@indexRange` attribute SHALL be present in the MPD and SHALL reference the byte range of the segment index in the track file. The value SHALL be formatted as a `byte-range-spec` as defined in [[!RFC7233]], referencing a single range of bytes.
+
+The `SegmentBase@timescale` attribute SHALL be present and its value SHALL match the value of the `timescale` field in the index segment.
+
+The `AdaptationSet@subsegmentStartsWithSAP` attribute SHALL be present in the MPD and SHALL have a value of `1` or `2`, depending on the sample structure of the [=media segments=].
+
+Issue: We need to clarify how to determine the right value. [#235](https://github.com/Dash-Industry-Forum/DASH-IF-IOP/issues/235)
+
+The `SegmentBase/Initialization@range` attribute SHALL reference the byte range of the initialization segment in the track file. The value SHALL be formatted as a `byte-range-spec` as defined in [[!RFC7233]], referencing a single range of bytes. The `Initialization@sourceURL` attribute SHALL NOT be used.
+
+<div class="example">
+Below is an example of common usage of the indexed addressing mode.
+
+The example defines a timescale of 48000 units per second, with the period starting at position 8100 (or 0.16875 seconds) on the [=sample timeline=]. The client can use the index segment referenced by `indexRange` to determine where the [=media segment=] containing position 8100 (and all other [=media segments=]) can be found. The byte range of the initialization segment is also referenced.
+
+<xmp highlight="xml">
+<MPD xmlns="urn:mpeg:dash:schema:mpd:2011">
+	<Period>
+		<AdaptationSet subsegmentStartsWithSAP="1">
+			<Representation>
+				<BaseURL>showreel_audio_dashinit.mp4</BaseURL>
+				<SegmentBase timescale="48000" presentationTimeOffset="8100" indexRange="848-999">
+					<Initialization range="0-847"/>
+				</SegmentBase>
+			</Representation>
+		</AdaptationSet>
+	</Period>
+</MPD>
+</xmp>
+
+Parts of the MPD structure that are not relevant for this chapter have been omitted - this is not a fully functional MPD file.
+</div>
+
+#### Structure of the index segment #### {#timing-addressing-indexed-indexstructure}
+
+The index segment SHALL consist of a single Segment Index Box (`sidx`) as defined by [[!ISOBMFF]]. The field layout is as follows:
```

Comment:
From @haudiobe: Why do we repeat the definition? At least there should be no SHALL

Reply:
The definition is repeated largely because I copied this text from 4.3, where it is repeated. I agree it felt a bit odd but I figured there must have been a reason for it if it were in 4.3.

So to develop the discussion further, I raise the question: why does this chapter exist in v4.3?